backend/src/tasks/hibp.ts

```diff
@@ -72,32 +96,34 @@ export const handler = async (commandOptions: CommandOptions) => {
   const services: Service[] = [];
   const vulns: Vulnerability[] = [];
   for (const domain of domainsWithIPs) {
+    console.log(domain.name);
     const results = await lookupEmails(breachesDict, domain);
-    console.log(
-      `Got ${Object.keys(results['Emails']).length} emails for domain ${
-        domain.name
-      }`
-    );
 
-    if (Object.keys(results['Emails']).length !== 0) {
-      vulns.push(
-        plainToClass(Vulnerability, {
-          domain: domain,
-          lastSeen: new Date(Date.now()),
-          title: 'Exposed Emails',
-          state: 'open',
-          source: 'hibp',
-          needsPopulation: false,
-          structuredData: {
-            emails: results['Emails'],
-            breaches: results['Breaches']
-          },
-          description: `Emails associated with ${domain.name} have been exposed in a breach.`
-        })
+    if (results) {
+      console.log(
+        `Got ${Object.keys(results['Emails']).length} emails for domain ${
+          domain.name
+        }`
       );
-      await saveVulnerabilitiesToDb(vulns, false);
-      console.log(results['Emails']);
-      console.log(results['Breaches']);
+
+      if (Object.keys(results['Emails']).length !== 0) {
+        vulns.push(
+          plainToClass(Vulnerability, {
+            domain: domain,
+            lastSeen: new Date(Date.now()),
+            title: 'Exposed Emails',
+            state: 'open',
+            source: 'hibp',
+            needsPopulation: false,
+            structuredData: {
+              emails: results['Emails'],
+              breaches: results['Breaches']
+            },
+            description: `Emails associated with ${domain.name} have been exposed in a breach.`
+          })
+        );
+        await saveVulnerabilitiesToDb(vulns, false);
+      }
```

Comment:
Let's add a console.error statement if results is equal to null (because this means we got an error).

Reply:
Do we want multiple console.error statements for the same thing, the comment above this has a console.error statement and then sets it to null, so wouldn't we have two error logs for the same error?

Reply:
Oh, I see, you're right, we only want a single console.error statement.